### PR TITLE
Remove tab/TabName usage where it's not needed

### PR DESCRIPTION
**Pull request by lafriks**
`tab` query argument and `TabName` in context is used only in profile so remove it from all other places where it's not used anymore.

### CODE REVIEW

**Review by 6543 — APPROVED** [review]

**Review by Gusted — APPROVED** [review]

**Comment by wxiaoguang** [comment]
> `tab` query argument and `TabPage` in context is used only in profile so remove it from all other places where it's not used anymore.

Do you mean `TabName`? I can see it set in:

```go
// SetDefaultParams sets common pagination params that are often used
func (p *Pagination) SetDefaultParams(ctx *Context) {
	p.AddParam(ctx, "sort", "SortType")
	p.AddParam(ctx, "q", "Keyword")
	p.AddParam(ctx, "tab", "TabName")
	// do not add any more uncommon params here!
	p.AddParam(ctx, "t", "queryType")
}

```

and used by:

image


Not sure whether it's used or not by `explore` (I haven't tested yet 😂)

File: templates/explore/code.tmpl [hunk 1]
```diff
@@ -3,7 +3,6 @@
 	{{template "explore/navbar" .}}
 	<div class="ui container">
 		<form class="ui form ignore-dirty" style="max-width: 100%">
-			<input type="hidden" name="tab" value="{{$.TabName}}">
```
Comment (delvh) [inline templates/explore/code.tmpl:6]: The fact that it was explicitly sent with the form perplexes me:
Why would it be added if it was unneeded?
Or is it **now** unneeded?

**Review by wxiaoguang — APPROVED** [review]
Hmm, this PR could be correct.

The `SetDefaultParams` also read the `TabName` from context data. Now it's refactored, so no problem

**Review by delvh — APPROVED** [review]

File: templates/explore/code.tmpl (hunk 1 above)
Comment (wxiaoguang) [reply templates/explore/code.tmpl:6]: In long time, old code -> rewriting -> old code -> rewriting.

Then some mechanism was removed, some dead code is left there.

Comment (lafriks) [reply templates/explore/code.tmpl:6]: yes, it's not used in explore page anymore, just some leftover after some other refactoring

**Comment by wxiaoguang** [comment]
Sorry for bothering, but the code symbol is `TabName`, not `TabPage`, what does `TabPage` refer to? Did I misunderstand something ....

**Pull request merged**